### PR TITLE
Three fixes for basic errors

### PR DESCRIPTION
**Pull request by Linchy**
Some errors I encountered while trying to use the develop build today.

### CODE REVIEW

File: Tools/Pipeline/Controls/PropertyGridTable.cs [hunk 1]
```diff
@@ -196,7 +196,7 @@ private void Drawable_Paint(object sender, PaintEventArgs e)
             if(_edit)
             {
                 if (!Global.Unix)
-                    _selectedCell.Edit(pixel1);
+                    _selectedCell?.Edit(pixel1);
```
Comment (cra0zy) [inline Tools/Pipeline/Controls/PropertyGridTable.cs]: The null check here is pointless, _selectedCell in this part of the code cannot be null because _edit will only get set if _selectedCell is not null.

https://github.com/Linchy/MonoGame/blob/016a5abf3e0651cb9aaa9a81d44dfd807117f180/Tools/Pipeline/Controls/PropertyGridTable.cs#L237

File: Tools/Pipeline/Controls/PropertyCells/CellRefs.cs [hunk 2]
```diff
@@ -3,27 +3,31 @@
...
     {
-        public CellRefs(string category, string name, object value, EventHandler eventHandler) : base(category, name, value, eventHandler)
+        public CellRefs(string category, string name, object value, EventHandler eventHandler)
+            : base(category, name, value, eventHandler)
         {
             if (Value == null)
                 Value = new List<string>();
 
-            DisplayValue = (Value as List<string>).Count > 0 ? "Collection" : "None";
+            DisplayValue = (Value as ICollection).Count > 0 ? "Collection" : "None";
```
Comment (cra0zy) [inline Tools/Pipeline/Controls/PropertyCells/CellRefs.cs]: Why the change to ICollection?

File: Tools/Pipeline/Controls/PropertyGridTable.cs (hunk 1 above)
Comment (Tzenchor) [inline Tools/Pipeline/Controls/PropertyGridTable.cs]: Also that is a characteristic of C# 6.0, but the build bot is using the C# 5.0 compiler

File: Tools/Pipeline/Controls/PropertyCells/CellRefs.cs (hunk 2 above)
Comment (Linchy) [inline Tools/Pipeline/Controls/PropertyCells/CellRefs.cs]: I can't actually manage to replicate the problem now as I've been changing a bunch of things with my project to get stuff working again, but at some point Value was being passed in as a Collection<SkinInfo> somehow, and not having a full understanding of the source I simply changed  it to support collections and it worked.

File: Tools/Pipeline/Controls/PropertyGridTable.cs (hunk 1 above)
Comment (Linchy) [inline Tools/Pipeline/Controls/PropertyGridTable.cs]: @cra0zy a screenshot of it happening: http://imgur.com/HQNNATo

This happens every time I open a `mgcb` file, then click on the root node, scroll down to references,  and double click `collection`.

Comment (cra0zy) [inline Tools/Pipeline/Controls/PropertyGridTable.cs]: Oh, I know what the problem is...

Comment (cra0zy) [inline Tools/Pipeline/Controls/PropertyGridTable.cs]: #4866

**Comment by KonajuGames** [comment]
In future please keep unrelated changes in separate PRs.  I have merged #4866, so please update this PR.

File: Tools/Pipeline/Controls/PropertyCells/CellRefs.cs (hunk 2 above)
Comment (cra0zy) [inline Tools/Pipeline/Controls/PropertyCells/CellRefs.cs]: Undo the List > ICollection change, it seems pointless...

**Comment by Linchy** [comment]
I'll re-open a new PR.
